docs/sources/community/getting-in-touch.md

```diff
@@ -5,16 +5,12 @@ title: Contacting the Loki Team
 
 For questions regarding Loki:
 
-- Open source Loki users are welcome to post technical questions on the Grafana Labs Community Forums under the Grafana Loki category at [community.grafana.com](https://community.grafana.com). Please be mindful that this is a community-driven support channel moderated by Grafana Labs staff where Loki maintainers and community members answer questions when bandwidth allows. Be sure to review the [Community Guidelines](https://community.grafana.com/guidelines) before posting. 
-- Users deploying Loki via [Grafana Cloud](https://grafana.com/products/cloud/) can submit support tickets via the [Grafana.com Account Portal](https://grafana.com/login). 
+- Open source Loki users are welcome to post technical questions on the Grafana Labs Community Forums under the Grafana Loki category at [community.grafana.com](https://community.grafana.com). Please be mindful that this is a community-driven support channel moderated by Grafana Labs staff where Loki maintainers and community members answer questions when bandwidth allows. Be sure to review the [Community Guidelines](https://community.grafana.com/guidelines) before posting.
+- Users deploying Loki via [Grafana Cloud](https://grafana.com/products/cloud/) can submit support tickets via the [Grafana.com Account Portal](https://grafana.com/login).
 - For questions regarding Enterprise support for Loki, you can get in touch with the Grafana Labs team [here](https://grafana.com/contact?pg=docs).
 
 Your feedback is always welcome! To submit feedback or a report a potential bug:
 
-- [File a GitHub issue](https://github.com/grafana/loki/issues/new) for bugs, issues and feature suggestions. Grafana UI issues for the Loki data source plugin should be posted directly to the [Grafana repository](https://github.com/grafana/grafana/issues/new), not the Loki repository. 
+- [File a GitHub issue](https://github.com/grafana/loki/issues/new) for bugs, issues and feature suggestions. Grafana UI issues for the Loki data source plugin should be posted directly to the [Grafana repository](https://github.com/grafana/grafana/issues/new), not the Loki repository.
 - Send an email to [lokiproject@googlegroups.com](mailto:lokiproject@googlegroups.com), or visit the [google groups](https://groups.google.com/forum/#!forum/lokiproject) page.
-- Join us for our monthly [Loki Community Call](https://docs.google.com/document/d/1MNjiHQxwFukm2J4NJRWyRgRIiK7VpokYyATzJ5ce-O8/edit?usp=sharing) on first Thursday of every month 12:00 UTC (8:00 am EST). 
-
-
-
-
+- Join us for our monthly [Loki Community Call](https://docs.google.com/document/d/1MNjiHQxwFukm2J4NJRWyRgRIiK7VpokYyATzJ5ce-O8/edit?usp=sharing) on first Thursday of every month alternate between EU(12:00 UTC) and US(17:00 UTC) timezones. Subscribe to the calendar in the doc for being up to date.
```

Comment:
Please double check with a native speaker, but I think this should be "alternating":

```suggestion
- Join us for our monthly [Loki Community Call](https://docs.google.com/document/d/1MNjiHQxwFukm2J4NJRWyRgRIiK7VpokYyATzJ5ce-O8/edit?usp=sharing) on first Thursday of every month alternating between EU (12:00 UTC) and US (17:00 UTC) timezones. Subscribe to the calendar in the doc for being up to date.
```